sdk/textanalytics/Azure.AI.TextAnalytics/src/Generated/Models/KeyPhraseExtractionTasksItem.cs

```diff
@@ -13,33 +13,5 @@ namespace Azure.AI.TextAnalytics
     /// <summary> The TasksStateTasksKeyPhraseExtractionTasksItem. </summary>
     public partial class KeyPhraseExtractionTasksItem : TaskState
     {
-        /// <summary> Initializes a new instance of KeyPhraseExtractionTasksItem. </summary>
-        /// <param name="lastUpdateDateTime"> . </param>
-        /// <param name="name"> . </param>
-        /// <param name="status"> . </param>
-        /// <exception cref="ArgumentNullException"> <paramref name="name"/> is null. </exception>
-        internal KeyPhraseExtractionTasksItem(DateTimeOffset lastUpdateDateTime, string name, JobStatus status) : base(lastUpdateDateTime, name, status)
```

Comment:
is the reason to delete this just because name is null? 
Name is required in the swagger which is why codegen is creating this constructor. Is the service not returning the name property?
If it is a bug, it will be better to do a transformation of the swagger for this preview, instead of adding more customized code. The less customize code we have to maintain, the better and easier it is long term.
Example: https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/autorest.md#make-generated-models-internal-by-default

Reply:
I am not sure how many times can we update swagger ? We already have few PRs open